webservices/partition/base.py

```diff
@@ -105,7 +105,7 @@ def create_constraints(cls, child, cycle):
         start, stop = cycle - 1, cycle
         cmds = [
             'alter table {child} alter column {primary} set not null',
-            'alter table {child} alter column load_date set not null',
+            'alter table {child} alter column filing_form set not null',
```

Comment:
Yup, I'm guessing the FEC must have a good reason to enforce that constraint, interestingly it's the only column with that constraint, besides the PK.